submissions/israelOladosu/htmlForms/ContactForm.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
```

Comment:
Well done!
You need to separate  the content of both pull request for [7] and [8]

You can't have 2 contact files.